### PR TITLE
Add more services for nfts and defi

### PR DESCRIPTION
**Pull request by kjhman21**

### CODE REVIEW

**Comment by kjhman21** [comment]
@yongchand I did something wrong with your master branch. I re-uploaded this in my own branch. Sorry for that.

**Comment by yongchand** [comment]
@kjhman21 Totally fine. Though a better PR title might be helpful to users?

**Comment by kjhman21** [comment]
@yongchand thanks! Updated.

File: Services.md [hunk 1]
```diff
@@ -27,3 +33,8 @@ This page has information about public services running on Klaytn.
     * [nBlocks](https://nblocks.io/)
   * Interchain Bridge
     * [OrbitBridge](https://bridge.orbitchain.io/)
+  * Statistics
+    * [Defillama](https://defillama.com/chain/Klaytn)
```
Comment (kjhman21) [inline Services.md:37]:
```suggestion
    * [Defi Llama](https://defillama.com/chain/Klaytn)
```

File: Services.md [hunk 2]
```diff
@@ -27,3 +33,8 @@ This page has information about public services running on Klaytn.
...
+    * [Klaylabs](https://www.klaylabs.net/)
```
Comment (kjhman21) [inline Services.md:38]:
```suggestion
    * [KLAYLABS](https://www.klaylabs.net/)
```

**Pull request merged**